src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -227,10 +227,13 @@ class SlickDataAccess(databaseConfig: Config, val dataAccess: DataAccessComponen
     // Insert an execution row
       executionInsert <- dataAccess.executionsAutoInc +=
         new Execution(
-          workflowExecution.workflowExecutionId.get,
-          key.scope.fullyQualifiedName,
-          key.index.fromIndex,
-          ExecutionStatus.NotStarted.toString)
+          workflowExecutionId = workflowExecution.workflowExecutionId.get,
+          callFqn = key.scope.fullyQualifiedName,
+          index = key.index.fromIndex,
+          status = ExecutionStatus.NotStarted.toString,
+          rc = None,
+          startDt = new Date().toTimestamp,
```

Comment:
Actually no I don't think this is ok.  We dump executions in the database potentially long before we even attempt to execute them.  

I'll make the start date nullable and null to start, then figure out how to update the start date when the call actually begins executing.